web/cobrands/sass/_base.scss

```diff
@@ -2499,8 +2499,16 @@ label .muted {
 }
 
 a#geolocate_link {
-  display: inline-block;
-  padding: 0.5em;
+  display: inline-block !important;
+  border-radius: 6px;
+  border: 1px solid #333333;
```

Comment:
I wonder whether we should use some existing Sass variables for this border colour and the text color on the line below? Or take the opportunity to define some new variables just for this?

Also, if we’re styling this geolocation button as a regular button, should it use our `button-size()` and/or `button-variant()` mixins instead?